test/src/e2e/informer.test.ts

```diff
@@ -94,10 +94,61 @@ describe("Informer", function() {
             ) {
                 await wait(500);
             }
-            subscription.addEventListener("message", (message: any) => {
+            subscription.once("message", (message: any) => {
                 expect(message).to.be.include(JSON.stringify(message2));
             });
         });
+
+        it("De-registration", async function() {
+            const subscription = new WebSocket(
+                `ws://localhost:${nodeA.informerPort}`
+            );
+            subscription.once("open", () => {
+                subscription.send(
+                    JSON.stringify({
+                        jsonrpc: "2.0",
+                        id: 1,
+                        method: "register",
+                        params: ["PeerAdded"]
+                    })
+                );
+            });
+            let subscribtionId: string;
+            subscription.once("message", (message: string) => {
+                subscribtionId = message.substr(26, 20);
```

Comment:
Could you extract the subscriptionId using `JSON.parse`?
Using `substr` is hard to mange.